javalin/src/main/java/io/javalin/plugin/rendering/vue/JavalinVue.kt

```diff
@@ -103,3 +103,14 @@ object PathMaster {
 }
 
 fun Path.readText() = String(Files.readAllBytes(this))
+
+fun escape(string: String?) = string?.toCharArray()?.map {
+    when (it) {
+        '<' -> "&lt;"
+        '>' -> "&gt;"
+        '&' -> "&amp;"
+        '"' -> "&quot;"
+        '\'' -> "&#x27;"
+        else -> it
+    }
```

Comment:
Any reason you want to do this escaping by hand instead of using a lib (for HTML escaping)?
- asked by a guy that doesn't do a lot of HTML escaping

Reply:
I don't really want to add a dependency to Javalin just for JavalinVue (it's used by a very small percentage of the user base), and these are the only values listed by OWASP.

Reply:
That sounds good, then 👍

Reply:
I believe you forgot one `' --> &#x27;` as referenced here https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html